encrypted-media-respec.html

```diff
@@ -1664,12 +1664,7 @@
                             <li><p>Clear all stored session data associated with this object, including the <a def-id="sessionId"></a> and license destruction record.</p>
                               <p class="note">A subsequent call to <a def-id="load"></a> with the value of this object's <a def-id="sessionId"></a> would fail because there is no data stored for that session ID.</p>
                             </li>
-                            <li>
-                              <p>
-                                Run the <a def-id="update-key-statuses-algorithm"></a> algorithm on the <var>session</var>, providing an empty sequence.
-                              </p>
-                            </li>
-                            <li><p>Run the <a def-id="session-close-algorithm"></a> algorithm on this object.</p></li>
+                            <li><p>Run the <a def-id="close-session-algorithm"></a> algorithm on this object.</p></li>
```

Comment:
I left this as _tbd_ under one of the other items in the OP.

There is a notion of "_closing a session_" (as distinct from closing a `MediaKeySession`) and it is also distinct from "_clearing all session data_". _Closing a session_ does not remove the license destruction record or record of key usage and _clearly all session data_ does not _close the session_ (or, at least, we do not say it does). Here, I think we want to clear all the session data and then _close the session_ and then perform the _Session Closed_ steps. _Close Session_ does exactly those last two things.